package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@apollo/client": "^3.3.21",
     "@expo-google-fonts/montserrat": "^0.2.0",
+    "@expo-google-fonts/nokora": "^0.2.0",
```

Comment:
Hmm let's stick with just montserrat because if we download a second font, it's gonna make our app size bigger. We need to update designs to reflect this